virtual_keyboard/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <main>
+    <section class="first-row row">
+      <button class="backtick dark-blue"><span>~</span> `</button>
+      <button><span>!</span> 1</button>
+      <button><span>@</span> 2</button>
+      <button><span>#</span> 3</button>
+      <button><span>$</span> 4</button>
+      <button><span>%</span> 5</button>
+      <button><span>:</span> 6</button>
+      <button><span>?</span> 7</button>
+      <button><span>*</span> 8</button>
+      <button><span>(</span> 9</button>
+      <button><span>)</span> 0</button>
+      <button><span>-</span> -</button>
+      <button><span>+</span> =</button>
+      <button class="backspace dark-blue">Backspace</button>
+    </section>
+    <section class="second-row row">
+      <button class="tab dark-blue">Tab</button>
+      <button>Q</button>
+      <button>W</button>
+      <button>E</button>
+      <button>R</button>
+      <button>T</button>
+      <button>Y</button>
+      <button>U</button>
+      <button>I</button>
+      <button>O</button>
+      <button>P</button>
+      <button>[</button>
+      <button>]</button>
+      <button><span>/</span> \</button>
+      <button>DEL</button>
```

Comment:
color of button not relevant to task. compare for all buttons

Reply:
Color is not provided in the task and I've used color picker.

Reply:
extension

Reply:
DEL should have the same background color, as Shift. Compare which buttons not relevant to their b-color in task and improve.